### ISSUE TITLE
Since the last update, i can't retrieve the sources anymore

### ISSUE THREAD

**Issue by staikfrite** [halcyon-tech/vscode-ibmi#879]
Issue text goes here.

* QCCSID: 65535
* Features:
   * git: false
   * grep: false
   * tn5250: false
   * setccsid: true
   * GENCMDXML.PGM: true
   * QZDFMDB2.PGM: true
   * iconv: true
   * attr: true
* SQL enabled: false
* Source dates enabled: false

Variants
```json
{
  "american": "%23@$",
  "local": "%23§$"
}
```

Errors:
```json
[
  {
    "command": "system \"CRTLIB LIB(ILEDITOR) TEXT('Code for i temporary objects. May be cleared.')\"",
    "code": 255,
    "stderr": "CPF2111: Bibliothek ILEDITOR bereits vorhanden.",
    "cwd": "/home/[MyLib]"
  },
  {
    "command": "system \"CHKOBJ OBJ(QSYS/QCPTOIMPF) OBJTYPE(*DTAARA)\"",
    "code": 255,
    "stderr": "CPF9801: Objekt QCPTOIMPF in Bibliothek QSYS nicht gefunden.",
    "cwd": "/home/[MyLib]"
  },
  {
    "command": "system \"CHKOBJ OBJ(QSYS/QCPFRMIMPF) OBJTYPE(*DTAARA)\"",
    "code": 255,
    "stderr": "CPF9801: Objekt QCPFRMIMPF in Bibliothek QSYS nicht gefunden.",
    "cwd": "/home/[MyLib]"
  }
]
```

**Comment by worksofliam** [comment]
Please share the contents of your Code for IBM i output channel after the list doesn't appear.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/3708366/178739532-03c8ad30-84de-4698-9904-65938f2ef651.png">

**Comment by staikfrite** [comment]
Sorry, but there is nothing to show in Code for IBM i as the file won't load.
FYI I'm on V7.4
![image](https://user-images.githubusercontent.com/43789601/191793619-dee08a44-cc11-4fcd-9b72-eb536fa110f8.png)

**Comment by worksofliam** [comment]
There is nothing in the Code for IBM i Output as shown in my screenshot?

**Comment by worksofliam** [comment]
What is the name of the object and library? And are you using an ASP? I see you have blurred it out but I am going to need to at information.

**Comment by worksofliam** [comment]
Closing as stale.

**Comment by staikfrite** [comment]
@worksofliam Hi, sorry I was working on other issues and also tried to find a fix on this by myself...

Unfortunately, I could not make any progress. The issue remains.
Sorry, for security purpose, I can't undisclose what I blured out, the library name and the file to edit name...

Code for IBM i Output does not show at all. Yes I'm using an ASP and I checked a rechecked everything, inclusive the
QSYS2.ASP_INFO => Ok.

I uninstalled and reinstalled alls the extensions. Removed the settings in %AppData%...

The errors messages I'm getting whilst reporting the issue as I did above are misleading: I created the missing Data Areas QCPTOIMPF and QCPFRMIMPF directly in QSYS and all I got was an error message after connecting to the server, that those are no longer necessary and should  be delete. Which I did... but without more success :-(
